cmd/ctr/commands/oci/oci.go

```diff
@@ -0,0 +1,51 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package oci
+
+import (
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+
+	"github.com/containerd/containerd/cmd/ctr/commands"
+	"github.com/containerd/containerd/containers"
+	"github.com/containerd/containerd/oci"
+)
+
+var Command = cli.Command{
+	Name:  "oci",
+	Usage: "OCI tools",
+	Subcommands: []cli.Command{
+		defaultSpecCommand,
+	},
+}
+
+var defaultSpecCommand = cli.Command{
+	Name:  "default-spec",
```

Comment:
Just "spec", so that we can add optional flags later?

Reply:
Updated